File: sk000801/입문/k의 개수.java
class Solution {
    public int solution(int i, int j, int k) {
        int answer = 0;
        for(int p = i; p <= j; p++) {
            answer += String.valueOf(p).length()-String.valueOf(p).replaceAll(String.valueOf(k), "").length();
        }
        return answer;
    }
}

File: kyw0716/level0/k의 개수.js
function solution(i, j, k) {
  let answer = 0;

  for (let index = i; index <= j; index++) {
    String(index)
      .split("")
      .forEach((v) => {
        if (v === `${k}`) answer += 1;
      });
  }

  return answer;
}

// https://school.programmers.co.kr/learn/courses/30/lessons/120887
